user_modules/shared_testcases/testCasesForRAI.lua

```diff
@@ -0,0 +1,233 @@
+local testCasesForRAI = {}
+
+require('atf.util')
+local events = require("events")
+
+--[[@Connecttest_onlywith_StartSDL_InitHMIOnReady: replace original connecttest with new one
+-- the uses only RunSDL and InitHMI
+--! @parameters:
+--! @FileName - new name of connecttest
+--]]
+function testCasesForRAI:Connecttest_onlywith_StartSDL_InitHMIOnReady(FileName)
```

Comment:
My opinion is all scripts that already use approach "generation user modules by code" must be refactored. 

Mentioned approach has next disadvantages comparing with approach with static module creation:
1. Copying and deleting of files during each test run (slow and required IO permissions)
2. Regular expression usage (very slow for 22,7 kb text files (connecttest.lua size), dependency on ATF code)
3. System violence (automatically generated code is strongly not recommended to use, especially on CI systems)

I agree with you in sentence "until we have stable common preconditions of ATF not using the ones from connecttest.lua, this is the easiest way" and that is why I propose use existed user_module with the same functionality, like user_modules/connecttest_resumption.lua or create static user_module in separated file if existed are not applicable for your purposes.